app/layout.tsx

```diff
@@ -69,35 +73,36 @@ const PageRoot = styled("div", {
 		/* ensure page content fills the view vertically */
 		min-height: 100svh;
 		grid-template-rows: auto auto auto 1fr;
-
-		/* design grid system */
 		display: grid;
 		grid-template-columns: var(--subgrid-columns);
 		--subgrid-columns: ${makeResponsiveGrid({
-			columnCount: 8,
+			columnCount: 10,
 			gutter: "20px",
-			margin: "30px",
-			scaleFully: true,
+			margin: "50px",
+			sourceDesignWidth: desktopDesignSize,
```

Comment:
since you're updating this, I'd recommend also pulling in the ftablet query to set this up for wide-tablet view